prompt-service/src/unstract/prompt_service/helper.py

```diff
@@ -333,9 +357,22 @@ def extract_table(
             "Unable to extract table details. "
             "Please contact admin to resolve this issue."
         )
+    if check_feature_flag_status(FeatureFlag.REMOTE_FILE_STORAGE):
+        fs_instance: FileStorage
+        if execution_source == ExecutionSource.IDE.value:
+            fs_instance = PromptServiceFileHelper.initialize_file_storage(
+                type=FileStorageType.PERMANENT
+            )
+        if execution_source == ExecutionSource.TOOL.value:
+            fs_instance = PromptServiceFileHelper.initialize_file_storage(
+                type=FileStorageType.TEMPORARY
+            )
```

Comment:
Same qury here that I raised for previous file

Reply:
Addressed